src/components/ProteinViewer/Popup/Variation/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { VariationDatum } from '@nightingale-elements/nightingale-variation/dist/nightingale-variation';
+import React from 'react';
+
+export type VariationDetail = {
+  feature?: VariationDatum;
+};
+type Props = {
+  detail: VariationDetail;
+};
+
+const ProtVistaVariationPopup = ({ detail }: Props) => {
+  if (!detail?.feature) return null;
+  const { accession, wildType, variant, start, size } = detail.feature;
+
+  const hasMultiple = size !== undefined && (size || 0) > 1;
+  return (
+    <section>
+      <h6>{accession}</h6>
+      <p>
+        {wildType} &gt; {variant}
```

Comment:
Usually variants are represented like `C>G` without space between characters.
Also I think it's important to show the consequence, but I now realise that `consequenceType` is not part an attribute of `VariationDatum`. I've added a suggestion on ebi-webcomponents/nightingale#272.

```suggestion
  const { accession, wildType, variant, start, size, consequenceType } = detail.feature;

  const hasMultiple = size !== undefined && (size || 0) > 1;
  return (
    <section>
      <h6>{accession}</h6>
      <p>
        {wildType}&gt;{variant} - {consequenceType}
```